app/process/handler/crypto-transfer/crypto-transfer.go

```diff
@@ -62,7 +61,7 @@ func NewCryptoTransferHandler(
 // Recover mechanism
 func (cth *CryptoTransferHandler) Recover(q *queue.Queue) {
 	cth.logger.Info("[Recovery] Executing Recovery mechanism for CryptoTransfer Handler.")
-	cth.logger.Infof("[Recovery] Database GET [%s] [%s] [%s] transactions.", txRepo.StatusInitial, txRepo.StatusSignatureSubmitted, txRepo.StatusEthTxSubmitted)
+	cth.logger.Infof("[Recovery] Database GET [%s] [%s] transactions.", txRepo.StatusInitial, txRepo.StatusSignatureSubmitted)
 
 	transactions, err := cth.transactionRepo.GetIncompleteTransactions()
```

Comment:
`GetIncompleteTransactions` will return `StatusInitial`, `StatusEthTxSubmitted` and `StatusSignatureSubmitted`. For every `StatusEthTxSubmitted` status we will do `checkForTransactionCompletion`. This must be fixed.

`GetIncompleteTransactions` must return only `StatusInitial` and `StatusSignatureSubmitted` statuses